lib/web/mage/adminhtml/browser.js

```diff
@@ -52,17 +53,21 @@ define([
                 content = '<div class="popup-window" id="' + windowId + '"></div>',
                 self = this;
 
-            if (this.modalLoaded === true &&
-                options &&
-                self.targetElementId &&
-                self.targetElementId === options.targetElementId
+            if (options
+                && self.targetElementId
+                && self.targetElementId === options.targetElementId
             ) {
                 if (typeof options.closed !== 'undefined') {
                     this.modal.modal('option', 'closed', options.closed);
                 }
                 this.modal.modal('openModal');
 
                 return;
+            } else if (_.isUndefined(options) && self.modalLoaded === true) {
```

Comment:
You can combine inner if with `(_.isUndefined(options) && self.modalLoaded === true)`